doc/using_azure-iot-sdk-c_with_the_latest_tls_version_only.md

```diff
@@ -0,0 +1,52 @@
+# How to configure the Azure IoT C SDK and currently supported TLS platforms to use TLS 1.2 only 
```

Comment:
Filename is to wordy. consider

configure_tls_client_protocal_version.md

Reply:
just tls.md looks good for me (other teams do that)

Reply:
https://github.com/MicrosoftDocs/azure-docs-pr/pull/100241/files  as example

Reply:
That name is not descriptive enough

Reply:
Makes sense. Taken.

Reply:
For .NET and Java we'll use the same name, so let us know what you came up with.

FWIW, the word is spelled *protocol*, so make sure you don't copy/paste that name.